workers/unity/Assets/Fps/Scripts/GameLogic/RandomNameGenerator.cs

```diff
@@ -0,0 +1,70 @@
+﻿using UnityEngine;
+
+namespace Fps
+{
+    public static class RandomNameGenerator
+    {
+        public static string GetName()
+        {
+            var prefix = Prefixes[Random.Range(0, Prefixes.Length)];
+            var postfix = Postfixes[Random.Range(0, Postfixes.Length)];
+
+            var timeOut = 200;
+            while (postfix.Length + prefix.Length > 15 && timeOut > 0)
+            {
+                postfix = Postfixes[Random.Range(0, Postfixes.Length)];
+                timeOut--;
+            }
+
+            var name = prefix + postfix;
+
+            if (timeOut == 0)
+            {
+                return name.Substring(0, 15);
+            }
+
+            if (name.Length < 13)
+            {
+                name += Random.Range(10, 99).ToString();
+            }
+
+            return name;
+        }
+
+
+        private static readonly string[] Prefixes =
+        {
+            "Angry",
+            "Splendid",
+            "Mega",
+            "Ugly",
+            "Noob",
+            "Dead",
+            "Dr",
+            "Cold",
+            "Fast",
+            "Good",
+            "Spark",
+            "Low",
+            "High",
```

Comment:
So uh.. this can produce `HighKid`... probably don't want that

Reply:
I think this class should be removed altogether before release...we won't be using it, AFAIK. I just used it to generate mock data for the results screen.

Reply:
Ah okay 👍

Reply:
I've added a TODO to remove this class, and there is a Jira ticket to do so (alongside tester classes)